src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,16 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: ['Under construction'],
```

Comment:
- [ ] Just as the first reviewer has said, It is a requirement that the initial state be an empty array. As per your implementation, the initial state is a string `under construction` which is wrong. Kindly make the initial state an empty array